<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFGIRCAppearance</class>
 <widget class="QWidget" name="CFGIRCAppearance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>470</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Default text color:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ColorButton" name="btnDefaultTextColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Channel action color:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ColorButton" name="btnChannelActionColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Network action color:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ColorButton" name="btnNetworkActionColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>CTCP color:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ColorButton" name="btnCtcpColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Error color:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ColorButton" name="btnErrorColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Background color:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="ColorButton" name="btnBackgroundColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>URL color:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="ColorButton" name="btnUrlColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Main font:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="FontButton" name="btnMainFont">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>User list</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Font:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="FontButton" name="btnUserListFont">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Selected text color:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ColorButton" name="btnUserListSelectedTextColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Selected background:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ColorButton" name="btnUserListSelectedBackgroundColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbTimestamps">
     <property name="toolTip">
      <string>Prepends all entries with [hh:mm:ss] timestamps.</string>
     </property>
     <property name="text">
      <string>Enable timestamps</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbWindowAlertOnImportantEvent">
     <property name="text">
      <string>Window/task bar alert on important event (private msg, callsign)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>gui/widgets/colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>FontButton</class>
   <extends>QPushButton</extends>
   <header>gui/widgets/fontbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnDefaultTextColor</tabstop>
  <tabstop>btnChannelActionColor</tabstop>
  <tabstop>btnNetworkActionColor</tabstop>
  <tabstop>btnCtcpColor</tabstop>
  <tabstop>btnErrorColor</tabstop>
  <tabstop>btnBackgroundColor</tabstop>
  <tabstop>btnUrlColor</tabstop>
  <tabstop>btnMainFont</tabstop>
  <tabstop>btnUserListFont</tabstop>
  <tabstop>btnUserListSelectedTextColor</tabstop>
  <tabstop>btnUserListSelectedBackgroundColor</tabstop>
  <tabstop>cbTimestamps</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
